bin/i18n/translation_status/translation_service.rb

```diff
@@ -8,10 +8,25 @@ def initialize
     # Normally we fallback to en-US, but we want to disable this so we detect missing strings.
     I18n.fallbacks.defaults = []
     I18n.backend.reload!
+    store_frontend_translations
   end
 
   # Returns true if a translation exists for a given key, otherwise returns false.
   def translated?(locale, key)
     I18n.exists?(key, locale: locale)
   end
+
+  # Aggregate all blockly-mooc translation JSON files to be loaded into i18n
+  def store_frontend_translations
+    locales_dir = Rails.root.join("../i18n/locales")
```

Comment:
I am not certain if this is the best way to traverse to the locales directory. Let me know if there is a better solution.

Reply:
Could you limit it to `locales` which have a dash in the middle, e.g. `*-*`? I think this current solution might be adding the `original` and `source` locales.